kclvm/sema/src/resolver/tests.rs

```diff
@@ -79,3 +79,24 @@ fn test_resolve_program_fail() {
     assert_eq!(diag.messages.len(), 1);
     assert_eq!(diag.messages[0].message, "expect int, got {str:int(1)}");
 }
+
+#[test]
+fn test_resolve_program_cycle_reference_fail() {
+    let mut program = load_program(
+        &["./src/resolver/test_fail_data/cycle_reference/file1.k"],
+        None,
+    )
+    .unwrap();
+    let scope = resolve_program(&mut program);
+    let err_messages = [
+        "There is a circular import reference between module file1 and file2",
```

Comment:
how to report circle import error for file1->file2->file3->file1?

Reply:
> how to report circle import error for file1->file2->file3->file1?

There is a circular import reference between module file1 and file3